virttest/utils_misc.py

```diff
@@ -4695,3 +4695,147 @@ def get_sosreport(path=None, session=None, remote_ip=None, remote_pwd=None,
         if session:
             session.close()
         return host_path
+
+
+class FS(object):
```

Comment:
Suggest move those codes to separate file since utils_misc.py pile up too many logic,and hard to maintain.

Reply:
sure, I too have this concern. I will do it.